dash_agent/README.md

```diff
@@ -1,39 +1,174 @@
-<!--
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Dash Agent 
 
-For information about how to write a good package README, see the guide for
-[writing package pages](https://dart.dev/guides/libraries/writing-package-pages).
+Dash agent is a framework enabling you to create and publish agents in CommandDash marketplace. 
 
-For general information about developing packages, see the Dart guide for
-[creating packages](https://dart.dev/guides/libraries/create-library-packages)
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages).
--->
-
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+## Getting started
 
-## Features
+You don't need to manually install the package. As the package comes pre-installed with the agent project that you can create using [dash-cli](). If it's not already activated. You can do so by running the following command in the terminal:
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+```shell
+dart pub global activate dash-cli
+```
 
-## Getting started
+Now, run the below shared command in the terminal once again:
```

Comment:
After activating the CLI, run the command to create your agent project. 

Replace `{{agent}}` with the unique name of your agent.